app/controllers/api/feeds_controller.rb

```diff
@@ -1,8 +1,9 @@
-class Api::FeedsController < ApplicationController
-  skip_before_action :authenticate!
-  before_action :authenticate_feeds_token!
+class Api::FeedsController < Api::BaseController
```

Comment:
This controller (which returns _all_ "non default settings" Feeds in the system) isn't long for this world. 
 [PRX/dovetail-router](https://github.com/PRX/dovetail-router.prx.org/pull/260) is going to move to using the `auth/podcast/123/feeds` endpoints, which is more scalable.

I made some changes in here to consolidate the api_admin_token code... but this functions exactly the same as before.